src/api/forms/service.js

```diff
@@ -216,7 +219,11 @@ export async function updateFormMetadata(formId, formUpdate) {
     }
 
     /** @type {Partial<FormMetadataDocument>} */
-    const updatedForm = { ...formUpdate }
+    const updatedForm = {
+      ...formUpdate,
+      updatedAt: new Date(),
+      updatedBy: author
```

Comment:
Should we consolidate `updatedAt` and `updatedBy` into the `formMetadata.update()` method?

i.e. Apply the latest author/date in a single place